resources/event/event.controller.js

```diff
@@ -0,0 +1,29 @@
+const Event = require("./event.model");
+const ScrapeEvents = require("../../utils/webScraper");
+
+const addScrapedEvent = async () => {
+  const scrappedEventsArray = await ScrapeEvents;
+  await scrappedEventsArray.forEach(async (event) => {
+    const data = new Event({
+      name: event.title,
+      scrapedEventDate: event.date,
+      scrapedEventId: event.eventId,
+      scrapedEventLink: event.eventLink,
+      imageUrl: event.imageLink,
+      location: event.location,
+      price: event.price,
+    });
+    await data.save();
+  });
+  return scrappedEventsArray;
+};
+
+const findEvent = async () => {
+  const events = await Event.find();
+  return events;
+};
```

Comment:
This code here could be optimized and move to a single function

Reply:
@abidex4yemi This is a single function.

Can you elaborate further? Do you mean handling error cases etc?